tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -4247,22 +4252,24 @@ public function cleanupDatabaseGroups():void {
 		$previousServer = $this->currentServer;
 		$this->usingServer('LOCAL');
 		foreach ($this->createdGroups as $group => $groupData) {
-			if ($this->isTestingWithLdap()) {
-				$this->cleanupGroup((string)$group);
-			} else {
-				$this->graphContext->adminDeletesGroupWithGroupId(
-					$groupData['id']
-				);
+			if ($groupData["possibleToDelete"]) {
```

Comment:
I don't see anywhere that sets "possibleToDelete" to `true`
So the code in this block will not be executed?

Reply:
These properties will be set `true` initially while adding the created group by method `addGroupToCreatedGroupsList`
https://github.com/owncloud/ocis/blob/65213abcb4141bb36f9d5cd6de2e6f5b43016fab/tests/acceptance/features/bootstrap/Provisioning.php#L2592-L2597

Reply:
OK, that is existing code.